padl/transforms.py

```diff
@@ -2248,3 +2300,776 @@ def importdump(transform_or_module):
+
+        for child in self.edges[current_node]:
+            if isinstance(child.transform, Batchify):
+                raise SyntaxError("Two batchify in same path is not allowed")
+            elif isinstance(child.transform, Unbatchify):
+                unbatchify_found = True
+                if child not in unbatchify_node:
+                    unbatchify_node[child] = Node(Identity() - 'unbatch Marker')
+                forward_edges[current_node][unbatchify_node[child]] = self.edges[current_node][child]
+                forward_edges[unbatchify_node[child]][self.output_node] = (None, None)
+            elif child == self.output_node and unbatchify_found:
+                raise SyntaxError('If a path has unbatchify, all paths must contain unbatchify')
+            else:
+                forward_edges[current_node][child] = self.edges[current_node][child]
+                forward_edges, unbatchify_found = self._generate_forward_dict(child, forward_edges, unbatchify_node,
+                                                                              unbatchify_found=unbatchify_found)
+        return forward_edges, unbatchify_found
+
+    def _generate_postprocess_dict(self,
+                                   current_node=None,
+                                   postprocess_edges=None,
+                                   ):
+        if postprocess_edges is None:
+            postprocess_edges = defaultdict(dict)
+
+        if current_node is None:
+            unbatchify_nodes = [n_ for n_ in self.edges if isinstance(n_.transform, Unbatchify)]
+            for unbatch_node in unbatchify_nodes:
+                postprocess_edges[self.input_node][unbatch_node] = (None, None)
+                postprocess_edges = self._generate_postprocess_dict(unbatch_node, postprocess_edges)
+            return postprocess_edges
+
+        for child in self.edges[current_node]:
+            if isinstance(child.transform, Unbatchify):
+                raise SyntaxError("Two Unbatchify in same path is not allowed")
+            else:
+                postprocess_edges[current_node][child] = self.edges[current_node][child]
+                postprocess_edges = self._generate_postprocess_dict(child, postprocess_edges)
+        return postprocess_edges
+
+    def _pd_splits(self, input_components=0):
+        """Generate splits
+
+        :param input_components:
+        :return:
+        """
+        preprocess_edges_dict, batchify_found = self._generate_preprocess_dict(self.input_node)
+        forward_edges_dict, unbatchify_found = self._generate_forward_dict()
+        postprocess_edges_dict = self._generate_postprocess_dict()
+
+        build_splits = []
+        if batchify_found:
+            self._preprocess_edges = preprocess_edges_dict
+            self._forward_edges = forward_edges_dict
+            build_splits += ['preprocess', 'forward']
+        else:
+            self._pd_preprocess = Compose([Identity() - 'Preprocess Identity'])
```

Comment:
I think you can use `identity` here instead of `Identity()` because `identity` is defined in this file.